source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+    gaea53              C5 head node
+    gaea54              C5 head node
+    gaea55              C5 head node
+    gaea56              C5 head node
+    gaea57              C5 head node
+    gaea58              C5 head node
+    gaea60              T6 Test access only
+    gaea61              C6 head node
+    gaea62              C6 head node
+    gaea63              C6 head node
+    gaea64              C6 head node
+    gaea65              C6 head node
+    gaea66              C6 head node
+    gaea67              C6 head node
+    gaea68              C6 head node
 
-     or
+    You will now be connected to NOAA RDHPCS: Gaea (CMRS/NCRC) C5 system.
+    To select a specific host, hit ^C within 5 seconds.
+    ^CEnter a hostname, or a unique portion of a hostname []:
 
-  .. code-block:: shell
+****************
+Data and storage
+****************
 
-     $ scontrol show partitions | grep dtn
+NFS filesystems
+===============
 
+Users and projects are given space on the :abbr:`NFS (Network File System)`.
+These locations are ideal for storing user and project applications,
+executables, and small data files.
 
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Area
+      - Path
+      - Permissions
+      - Quota
+      - Backups
+      - Purged
+      - On Compute Nodes
+    * - User Home
+      - :file:`/ncrc/home[12]/<userID>`
+      - User set
+      - 50 GB
+      - Yes
+      - No
+      - Yes
+    * - Project Home
+      - :file:`/ncrc/proj/<projID>`
+      - Project set
+      - 100 GB
+      - Yes
+      - No
+      - Yes
+
+
+GPFS filesystems
+================
+
+Each compute cluster, C5 and C6, have their own file system called F5 and F6
+respectively, mounted at :file:`/gpfs/f5` and :file:`/gpfs/f6`.
 
-Transferring Data to/from Gaea
-------------------------------
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Area
+      - Path
+      - Permissions
+      - Quota
+      - Backups
+      - Purged
+      - On compute nodes
+    * - F5 Member Work
+      - :file:`/gpfs/f5/<projID>/scratch/<userID>`
+      - User set
+      - N/A
+      - No
+      - No
+      - C5 only
+    * - F5 Project Work
+      - :file:`/gpfs/f5/<projID>/proj-shared`
+      - 770
+      - N/A
+      - No
+      - No
+      - C5 only
+    * - F5 World Work
+      - :file:`/gpfs/f5/<projID>/world-shared`
+      - 775
+      - N/A
+      - No
+      - No
+      - C5 only
+    * - F6 Member Work
+      - :file:`/gpfs/f6/<projID>/scratch/<userID>`
+      - User set
+      - N/A
+      - No
+      - No
+      - C6 only
+    * - F6 Project Work
+      - :file:`/gpfs/f6/<projID>/proj-shared`
+      - 770
+      - N/A
+      - No
+      - No
+      - C6 only
+    * - F6 World Work
+      - :file:`/gpfs/f6/<projID>/world-shared`
+      - 775
+      - N/A
+      - No
+      - No
+      - C6 only
+
+Move data to and from Gaea
+==========================
+
+The suggested way to move data to and from Gaea is to use `Globus Online
```

Comment:
Recommend:

The preferred way to move data...is Globus Online.

Reply:
Fixed in new push.